http/graphql/pom.xml

```diff
@@ -15,50 +15,5 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-smallrye-graphql</artifactId>
         </dependency>
-        <dependency>
-            <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-smallrye-opentracing</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>io.quarkus.qe</groupId>
-            <artifactId>quarkus-test-service-jaeger</artifactId>
-            <scope>test</scope>
-        </dependency>
     </dependencies>
-    <profiles>
```

Comment:
Why are the profiles removed?

Reply:
We can run tests on Windows as now there is no container. We don't need extension dependency here as it is not used.